views/default/js/scheduling/table.js

```diff
@@ -3,169 +3,74 @@ elgg.provide('elgg.scheduling');
 
 /**
  * Add a new column to each of the existing rows
- *
- * @param {Object} event
  * @return void
  */
-elgg.scheduling.addColumn = function(event) {
-	var rows = $('#elgg-table-scheduling tr');
-
-	// Count columns leaving out the first one with dates
-	var columnCount = $('#elgg-table-scheduling tr th').length -1;
-
-	rows.each(function(key, value) {
-		if (key === 0) {
-			$(this).append('<th>' + elgg.echo('scheduling:slot:title', [columnCount]) + '</th>');
-			return true;
+elgg.scheduling.addColumn = function () {
```

Comment:
I just realized that the plugin doesn't have support for Elgg 1.8 in the first place. So this might as well be an AMD module. But I'll merge this as is. Let's convert to AMD in a separate PR.